distribution/ddf-common/src/main/resources/etc/system.properties

```diff
@@ -158,4 +158,5 @@ karaf.secured.services=(&(osgi.command.scope=*)(osgi.command.function=*))
 #org.osgi.framework.security=osgi
 #org.osgi.framework.trust.repositories=${karaf.home}/etc/trustStore.ks
 ws-security.subject.cert.constraints = .*
-
+org.apache.cxf.configuration.jsse.sslAllowedAlgorithms = .*_EXPORT_.*, .*_WITH_AES_.*, .*_WITH_3DES_.*
+org.apache.cxf.configuration.jsse.sslDisallowedAlgorithms = .*_WITH_NULL_.*, .*_DH_anon_.*
```

Comment:
This should not be added. We have allowed and not allowed algorithms already. Further, these are the incorrect settings.